### PR TITLE
Fix: atlas attachment serving + broken test

### PR DESCRIPTION
**Pull request by lgalabru**
After investigation, it turns out the atlas test was broken for a good reason.
- Attachment serving broke with a recent refactoring
- The BNS pricing table changed and the value provided was too low, as a consequence the `namespace-preorder` contract call was failing.

### CODE REVIEW

File: src/net/atlas/db.rs [hunk 1]
```diff
@@ -387,7 +387,7 @@ impl AtlasDB {
         content_hash: &Hash160,
     ) -> Result<Option<Attachment>, db_error> {
         let hex_content_hash = to_hex(&content_hash.0[..]);
-        let qry = "SELECT content, hash FROM attachments WHERE hash = ?1 AND was_instantiated = 0"
+        let qry = "SELECT content, hash FROM attachments WHERE hash = ?1 AND was_instantiated = 1"
```
Comment (jcnelson) [inline src/net/atlas/db.rs:390]: There haven't been any attachments propagated to the mainnet network yet, right?  i.e. We're certain this change isn't going to break anything?

**Review by jcnelson — APPROVED** [review]
LGTM; just had one question I need some clarity on first.

File: src/net/atlas/db.rs (hunk 1 above)
Comment (lgalabru) [reply src/net/atlas/db.rs:390]: The BNS endpoints + registrar not being live at this point, the only interactions that happened with the contract were internal tests (https://explorer.stacks.co/txid/SP000000000000000000002Q6VF78.bns?chain=mainnet)

**Review by kantai — APPROVED** [review]

File: .github/workflows/bitcoin-tests.yml [hunk 2]
```diff
@@ -16,9 +16,6 @@ jobs:
           DOCKER_BUILDKIT: 1
         run: docker build -f ./.github/actions/bitcoin-int-tests/Dockerfile.bitcoin-tests .
   atlas-test:
-    # disable this job/test for now, since we haven't seen this pass
-    #  on github actions in a while
-    if: ${{ false }}
```
Comment (kantai) [inline .github/workflows/bitcoin-tests.yml:21]: 🥳

**Pull request merged**